lib/tasks/data/mapped_locations.yml

```diff
@@ -2,18 +2,71 @@
 - ['barking', 'barking and dagenham']
 - ['bedfordshire', 'central bedfordshire']
 - ['brighton', 'brighton and hove']
+- ['bromley, london', 'bromley']
 - ['burton', 'burton upon trent']
 - ['burton-on-trent', 'burton upon trent']
 - ['burton on trent', 'burton upon trent']
+- ['camden, london', 'camden']
+- ['camden town, london', 'camden']
 - ['chelsea', 'kensington and chelsea']
+- ['chelsea, london', 'kensington and chelsea']
+- ['city of london, london', 'city of london']
+- ['city of westminster, london', 'westminster']
 - ['cleveland', 'redcar and cleveland']
+- ['croydon, london', 'croydon']
 - ['dagenham', 'barking and dagenham']
+- ['ealing, london', 'ealing']
 - ['east cheshire', 'cheshire east']
 - ['east riding', 'east riding of yorkshire']
+- ['enfield, london', 'enfield']
 - ['fulham', 'hammersmith and fulham']
+- ['greenwich, london', 'greenwich']
+- ['hackney, london', 'hackney']
 - ['hammersmith', 'hammersmith and fulham']
+- ['hammersmith and fulham, london', 'hammersmith and fulham']
+- ['haringey, london', 'haringey']
 - ['herefordshire', 'herefordshire, county of']
+- ['islington, london', 'islington']
 - ['kensington', 'kensington and chelsea']
+- ['kensington and chelsea, london', 'kensington and chelsea']
+- ['kingston', 'kingston upon thames']
+- ['kingston-upon-thames', 'kingston upon thames']
+- ['lambeth, london', 'lambeth']
+- ['lewisham, london', 'lewisham']
+- ['london borough of barking and dagenham', 'barking and dagenham']
+- ['london borough of barnet', 'barnet']
+- ['london borough of bexley', 'bexley']
+- ['london borough of brent', 'brent']
+- ['london borough of bromley', 'bromley']
+- ['london borough of camden, london', 'camden']
+- ['london borough of croydon', 'croydon']
+- ['london borough of ealing', 'ealing']
+- ['london borough of enfield', 'enfield']
+- ['london borough of greenwich', 'greenwich']
+- ['london borough of hackney, london', 'hackney']
+- ['london borough of hammersmith and fulham, london', 'hammersmith and fulham']
+- ['london borough of haringey, london', 'haringey']
+- ['london borough of harrow', 'harrow']
+- ['london borough of havering', 'havering']
+- ['london borough of hillingdon', 'hillingdon']
+- ['london borough of hounslow', 'hounslow']
+- ['london borough of islington, london', 'islington']
+- ['london borough of kensington and chelsea, london', 'kensington and chelsea']
+- ['london borough of kingston upon thames', 'kingston upon thames']
+- ['london borough of lambeth, london', 'lambeth']
+- ['london borough of lewisham', 'lewisham']
+- ['london borough of merton', 'merton']
+- ['london borough of newham', 'newham']
+- ['london borough of redbridge', 'redbridge']
+- ['london borough of richmond upon thames', 'richmond upon thames']
+- ['london borough of southwark', 'southwark']
+- ['london borough of sutton', 'sutton']
+- ['london borough of tower hamlets', 'tower hamlets]
```

Comment:
Here it is: the closing quote is missing.

Reply:
Cheers, only just twigged that the automatic reviews ran every time I made a change and I could have used those to spot the missing punctuation :l

Reply:
Now you know!